readthedocs/projects/constants.py

```diff
@@ -355,3 +355,19 @@
 )
 
 GITHUB_PR_PULL_PATTERN = 'pull/{id}/head:external-{id}'
+
+SEARCH_ANALYTICS_PARAMS = {
+    'period': (
+        ('recent', 'Recent Queries'),
+        ('last-24-hrs', 'Top Queries of Past 24 Hrs'),
+        ('last-48-hrs', 'Top Queries of Past 48 Hrs'),
+        ('last-7-days', 'Top Queries of Past 7 Days'),
+        ('last-1-month', 'Top Queries of Past 1 Month'),
```

Comment:
Should we show a `last-3-months` option, if we are storing the data?

Reply:
Ohh yeah...
I forgot to update this part.
With the previous modelling, it was having some problems.
But I have changed it according to your suggestion.
I will update the PR

Reply:
Done.